Frameworks/UIKit.Xaml/ActivityIndicatorView.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<Grid
+    x:Class="UIKit.Xaml.ActivityIndicatorView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UIKit.Xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    x:Name="Root">
+    <!-- TODO: Update property names in path for binding once they have been updated in the header -->
+    <ProgressRing x:Name="progressRing"
+                  IsActive="{Binding ElementName=Root, Path=ActivityIndicatorView_IsActive, Mode=OneWay}"
+                  Foreground="{Binding ElementName=Root, Path=ActivityIndicatorView_Foreground, Mode=OneWay}"
+                  Height="{Binding ElementName=Root, Path=ActivityIndicatorView_Height, Mode=OneWay}"
+                  Width="{Binding ElementName=Root, Path=ActivityIndicatorView_Width, Mode=OneWay}"
```

Comment:
The frame and indicator have independent sizes. The user is able to resize the frame, but the indicator inside can only be set via the provided styles.

---
In reply to: [113538678](https://github.com/Microsoft/WinObjC/pull/2608#discussion_r113538678) [](ancestors = 113538678)